src/database/sqlite/sqlite_db_migration.c

```diff
@@ -494,6 +494,86 @@ static int do_ml_migration_v1_v2(sqlite3 *database)
     return 0;
 }
 
+// Actions for health migration
+const char *database_health_migrate_v0_v1[] = {
+    "CREATE TABLE IF NOT EXISTS alert_hash(hash_id blob PRIMARY KEY, date_updated int, alarm text, template text, "
+    "on_key text, class text, component text, type text, os text, hosts text, lookup text, "
+    "every text, units text, calc text, families text, plugin text, module text, charts text, green text, "
+    "red text, warn text, crit text, exec text, to_key text, info text, delay text, options text, "
+    "repeat text, host_labels text, p_db_lookup_dimensions text, p_db_lookup_method text, p_db_lookup_options int, "
+    "p_db_lookup_after int, p_db_lookup_before int, p_update_every int, source text, chart_labels text, "
+    "summary text, time_group_condition INT, time_group_value DOUBLE, dims_group INT, data_source INT)",
```

Comment:
What I wrote above about the null fields is also applied for each table here. Let us force the engine to do a double check, to be sure we will not have data loss.

Reply:
if `every` is our `update_every`, ideally, it should be integer